python/aswfdocker/data/versions.yaml

```diff
@@ -687,6 +688,8 @@ ci-packages:
     - "2022.1"
     - "2023.2"
     - "2024.1"
+  expat:
+    - "2024.1"
```

Comment:
I have no idea how the `ci-packages` section is sorted,  if at all. As the newest package, at the end is fine.